lib/tasks/refresh_allocation_data.rake

```diff
@@ -0,0 +1,6 @@
+namespace :allocations do
+  desc 'Refresh allocation moves count and status'
+  task refresh_data: :environment do
+    Allocation.all.each(&:refresh_status_and_moves_count!)
```

Comment:
maybe better to use `find_each` rather than `.all.each`, unless there is a good reason not to ?